### PR TITLE
@orta => Bump graphql/tools to reenable stitching and merging

### PR DESCRIPTION
**Pull request by mzikherman**
Locally this now works with Convection stitching/merging (and feature flag enabled).

### CODE REVIEW

**Comment by orta** [comment]
Yep, looks 👍 to me

File: src/lib/mergeSchemas.js [hunk 1]
```diff
@@ -77,4 +77,6 @@ export async function mergeSchemas() {
       },
     }),
   })
+  mergedSchema.__allowedLegacyNames = ["__id"]
```
Comment (mzikherman) [inline src/lib/mergeSchemas.js]: After tracing thru some of the `buildSchema`, etc. methods used above, this was the one that wound up working.

Which makes sense, the result of this function _is_ the schema that we are operating on, executing resolvers, etc. So this is the schema that needs this option set.

**Comment by orta** [comment]
Rock

File: src/lib/mergeSchemas.js (hunk 1 above)
Comment (damassi) [reply src/lib/mergeSchemas.js]: @alloy should be able to comment more about this as he's done some recent relay work around this option

**Pull request merged**